cmd/manager/main.go

```diff
@@ -129,6 +143,7 @@ func main() {
 		WithPath("/metrics").
 		WithPort(strconv.Itoa(metricsPort)).
 		WithServiceMonitor().
+		WithCollectors(metrics.GetMetricsAggregator().GetMetrics()).
```

Comment:
This part looks unrelated to the topic, and nontrivial.

Reply:
I can move it into another PR but basically it was an oversight from the first refactoring I did in this repo. I can move it into another PR.